README.md

```diff
@@ -57,24 +73,29 @@ worker.get('/myResource.html', function(request, response) {
 });
 ```
 
-## Why I would like to support more than just handling requests?
+TODO: a working code example that can be executed to see how 'Your code' would actually look like
+
+### onInstall(?) TODO
+### onActivate(?) TODO
```

Comment:
Called just after `onInstall` or each time that you update your ServiceWorker, useful to update caches or any part of your application logic.